doc/particles.md

```diff
@@ -241,55 +216,56 @@ desired positioning.
 game.add(
   ParticleComponent(
     particle: CircleParticle(
-      radius: game.size.x / 2,
+      radius: game.size.width / 2,
       paint: Paint()..color = Colors.red.withOpacity(.5),
-    )
-  )
+    ),
+  ),
 );
 ```
 
-## SpriteParticle
+## Sprite Particle
```

Comment:
I think it is better to keep the name as it is in the code? I think we do that in the rest of the docs now.